src/Command/CreateCommand.php

```diff
@@ -80,7 +81,8 @@ protected function configure(): void
             ->addOption('fields', 'f', InputOption::VALUE_OPTIONAL, 'Table fields to generate.')
             ->addOption('table-comment', null, InputOption::VALUE_OPTIONAL, 'Table comment.')
             ->addOption('and', null, InputOption::VALUE_OPTIONAL, 'And junction.')
-            ->addOption('namespace', null, InputOption::VALUE_OPTIONAL, 'Migration file namespace.')
+            ->addOption('path', null, InputOption::VALUE_REQUIRED, 'Path to migration directory.')
```

Comment:
They should not be optional, what happens if I configure the paths in the configurations, I should execute the commands without these options.